tools/ovc/openvino/tools/ovc/__init__.py

```diff
@@ -18,3 +19,9 @@
 if is_optimum() and optimum_version is not None:
     telemetry = init_ovc_telemetry("Optimum Intel", optimum_version)
     telemetry.send_event("optimum", "import", "import_from_optimum,ov_version:{}".format(get_rt_version()))
+
+if is_torch() and 'torch' in sys.modules:
+    from openvino.runtime import get_version as get_rt_version  # pylint: disable=no-name-in-module,import-error
```

Comment:
```suggestion
    from openvino import get_version as get_rt_version  # pylint: disable=no-name-in-module,import-error
```

Reply:
Removed this import, as it is redundant and applied this comment in the import above.